delta-api/src/main/java/io/cdap/delta/api/DDLEvent.java

```diff
@@ -79,7 +79,7 @@ public String toString() {
     if (primaryKey != null && !primaryKey.isEmpty()) {
       stringJoiner.add("primaryKey=" + primaryKey);
     }
-    if (this.getOffset().toString() != null) {
+    if (!this.getOffset().toString().isEmpty()) {
```

Comment:
We should not write logic on return value of toString(), it's fine to just check if getOffset is not null